components/expenses/ExpenseQueue.tsx

```diff
@@ -50,6 +53,10 @@ export default function ExpenseQueue({ className }: ExpenseQueueProps) {
     refetch();
   };
 
+  const handleSyncQueue = () => {
+  syncExpenseQueueMutation.mutate();
```

Comment:
take a look at AutoSyncManager.tsx , it has some extra protections to make sure the mutation isn't currently in progress (sync in progress) which you might want to add to avoid a race condition